src/themes/teams/components/Input/inputStyles.ts

```diff
@@ -26,11 +26,23 @@ const inputStyles: ComponentSlotStylesInput<InputProps, InputVariables> = {
       borderColor: v.inputFocusBorderColor,
       borderRadius: v.inputFocusBorderRadius,
     },
+    ...(p.iconPosition === 'start' &&
```

Comment:
nit: we could simplify and decouple conditions in the following way:

```js
...
...(p.clearable && { padding: v.inputPaddingWithIconAtEnd, }),
...(p.icon && { padding: p.iconPosition === 'start'
  ? v.inputPaddingWithIconAtStart
  : v.inputPaddingWithIconAtEnd
```

Most important benefit of this approach would be decoupling the effects of `clearable` and `iconPosition` props for styles, so that if, say, one of these props will be eliminated in future, it would be just a matter of removing the corresponding paragraph

Reply:
Rearranged styles and changed variables names. Thanks!